### ISSUE TITLE
pandoc + xelatex chocking on the \tightlist tex command.

### ISSUE THREAD

**Issue by antoine-lizee** [ReScience/ReScience-submission#9]
I get a

```
! Undefined control sequence.
l.227 \tightlist
```

which apparently comes from a "new" feature of [pandoc](https://github.com/jgm/pandoc/pull/1571) that has not been integrated in the template. I will propose a PR modifying the template, but it asks the bigger question of maybe forking the default [latex template](https://github.com/jgm/pandoc-templates) to integrate the changes these guys make along the way.

**Comment by rougier** [comment]
The current latex template in ReScience could certainly be improved. I removed a lot of if/then/else statements mostly because I was discovering the pandoc tool and I may have break future compatibility. 

What do you mean exactly by forking the latex template ? If you have a long-term solution in mind, that would be great.

**Comment by antoine-lizee** [comment]
The [pandoc default templates repo](https://github.com/jgm/pandoc-templates) could be forked to make the basis of the template we're working on. We modify it in the forms of branches for instances, and then [integrate](https://help.github.com/articles/syncing-a-fork/) the changes they make along the way.

**Comment by rougier** [comment]
But how do we link the ReScience-submission template to this branch then ?

**Comment by antoine-lizee** [comment]
Not sure. We could use a submodule, or just have a different repo for the template itself, that we replace in the submission repo when it get changed.

**Comment by ThomasA** [comment]
Fork it off the Pandoc templates repo under the ReScience account and 
then link that repo into the submission repo using git subtree?

**Comment by antoine-lizee** [comment]
Yep, I'd go the route of a separate repo for the template alone - just not convinced about the submodule thing, which tends to break/be slightly complicated to maintain. Copy-paste-commit could be a good start.

**Comment by rougier** [comment]
And we have also the `rmarkdown` alternative proposal. Maybe we should start some experiments in a dedicated repository.

**Comment by antoine-lizee** [comment]
+1
A branch might be enough though? That's what I did in my fork.

**Comment by eddelbuettel** [comment]
Where is the `rmarkdown` proposal?  I looked but failed to find it.

**Comment by antoine-lizee** [comment]
cf https://github.com/ReScience/ReScience/issues/16
